scrapy/templates/project/module/settings.py.tmpl

```diff
@@ -16,75 +16,75 @@ NEWSPIDER_MODULE = '$project_name.spiders'
 
 
 # Crawl responsibly by identifying yourself (and your website) on the user-agent
-#USER_AGENT = '$project_name (+http://www.yourdomain.com)'
+# USER_AGENT = '$project_name (+http://www.yourdomain.com)'
 
 # Obey robots.txt rules
 ROBOTSTXT_OBEY = True
 
 # Configure maximum concurrent requests performed by Scrapy (default: 16)
-#CONCURRENT_REQUESTS = 32
+# CONCURRENT_REQUESTS = 32
```

Comment:
I figured it might be intentional. The thing is that PEP8 linters raise many warnings because of that.

I have a PEP8 linter enabled by default, so that's where my pain came from. 😄